KPP/fullchem/fullchem_RateLawFuncs.F90

```diff
@@ -2078,82 +2074,10 @@ FUNCTION HOBrHetSO4H2( H ) RESULT( k )
-!-------------------------------------------------------------------------------
-! NOTE: These functions are for use with sulfur in-cloud chemistry
-! being computed in KPP.  Comment out for now. (bmy, 9/16/21)
-!
-!  FUNCTION HOBrUptkByHSO3m( H ) RESULT( k )
-!    !
-!    ! Computes the uptake rate [1/s] for the HOBr + HSO3(-) reaction.
-!    !
-!    TYPE(HetState), INTENT(IN) :: H              ! Hetchem State
-!    REAL(dp)                   :: k              ! rxn rate [1/s]
-!    !
-!    REAL(dp) :: brLiq,     gammaLiq,     k_HOBr_Cl
-!    REAL(dp) :: k_HOBr_Br, k_HOBr_HSO3m, k_HOBr_SO3mm
-!    REAL(dp) :: k_tot,     srMw
-!    !
-!    k        = 0.0_dp
-!    brLiq    = 0.0_dp
-!    gammaLiq = 0.0_dp
-!    srMw     = SR_MW(ind_HOBr)
-!    !
-!    IF ( .not. H%stratBox ) THEN
-!       !
-!       ! HOBr + HBr rxn probability in tropospheric liquid cloud
-!       CALL Gam_HOBr_CLD(                                                    &
-!            H,         gammaLiq,  k_tot,                                     &
-!            k_HOBr_Cl, k_HOBr_Br, k_HOBr_HSO3m, k_HOBr_SO3mm )
-!       !
-!       ! Branching ratio for liquid path of HOBr + HSO3m in cloud
-!       brLiq = k_HOBr_HSO3m / k_tot
-!       !
-!       ! Compute overall HOBr removal rate in cloud
-!       k = k + CloudHet( H, srMw, gammaLiq, 0.0_dp, brLiq, 0.0_dp )
-!       !
-!    ENDIF
-!
-!    ! Assume HOBr is limiting, so update the removal rate accordingly
-!    k = kIIR1Ltd( C(ind_HOBr), C(ind_HSO3m), k )
-!  END FUNCTION HOBrUptkByHSO3m
-!
-!  FUNCTION HOBrUptkBySO3mm( H ) RESULT( k )
-!    !
-!    ! Computes the uptake rate [1/s] for the HOBr + HSO3(-) reaction.
-!    !
-!    TYPE(HetState), INTENT(IN) :: H              ! Hetchem State
-!    REAL(dp)                   :: k              ! rxn rate [1/s]
-!    !
-!    REAL(dp) :: brLiq,     gammaLiq,     k_HOBr_Cl
-!    REAL(dp) :: k_HOBr_Br, k_HOBr_HSO3m, k_HOBr_SO3mm
-!    REAL(dp) :: k_tot,     srMw
-!    !
-!    k    = 0.0_dp
-!    srMw = SR_MW(ind_HOBr)
-!    !
-!    IF ( .not. H%stratBox ) THEN
-!       !
-!       ! HOBr + HBr rxn probability in tropospheric liquid cloud
-!       CALL Gam_HOBr_CLD(                                                    &
-!            H,         gammaLiq,  k_tot,                                     &
-!            k_HOBr_Cl, k_HOBr_Br, k_HOBr_HSO3m, k_HOBr_SO3mm )
-!       !
-!       ! Branching ratio for liquid path of HOBr + HSO3m in cloud
-!       brLiq = k_HOBr_SO3mm / k_tot
-!       !
-!       ! Compute overall HOBr removal rate in cloud
-!       k = k + CloudHet( H, srMw, gammaLiq, 0.0_dp, brLiq, 0.0_dp )
-!       !
-!    ENDIF
-!
-!    ! Assume HOBr is limiting, so update the removal rate accordingly
-!    k = kIIR1Ltd( C(ind_HOBr), C(ind_SO3mm), k )
-!  END FUNCTION HOBrUptkBySO3mm
-!-------------------------------------------------------------------------------
+
+    ! Assume HOBr is limiting, so update the removal rate accordingly
+    k = kIIR1Ltd( C(ind_HOBr), C(ind_SO3mm), k )
```

Comment:
These are some wild function names

Reply:
I know, a lot of those were in the original code, which I didn't touch.